### ISSUE TITLE
New version

### ISSUE THREAD

**Issue by shinzui** [mlms13/bs-decode#19]
Is it possible to release a new version? I am using master in various projects, but I would like to use a stable version across them.

**Comment by splodingsocks** [comment]
I heard that @mlms13 might be doing this today ;)

**Comment by mlms13** [comment]
Yeah, sorry for not responding sooner, but this issue has been very motivating for me! :grin: 

I'm going to throw together a couple Github Projects to track the next 2ish releases, and I really hope to make a hard stop on my `docs` branch this afternoon and just release whatever's done at that point.

**Comment by phated** [comment]
Hooray!  I won't need to publish a fork ;)
